bot_manager.py

```diff
@@ -21,14 +21,21 @@
 
 class BotManager:
 
-    def __init__(self, terminateEvent, callbackEvent, name, team, index, modulename):
+    def __init__(self, terminateEvent, callbackEvent, config_file, name, team, index, modulename):
         self.terminateEvent = terminateEvent
         self.callbackEvent = callbackEvent
+        self.config_file = config_file
         self.name = name
         self.team = team
         self.index = index
         self.module_name = modulename
 
+    def load_agent(self, agent_module):
+        try:
+            agent = agent_module.Agent(self.name, self.team, self.index, config_file=self.config_file)
```

Comment:
Having the parameter be called config_file seems confusing, because they're really just getting one section. I think it would be more intuitive to give it the same name as the corresponding section header.

Likewise for the BotManager constructor and anywhere else where the variable refers to only one section.